pycontrails/models/cocip/cocip.py

```diff
@@ -1310,9 +1319,20 @@ def process_met_datasets(
             "Specific humidity enhancement of the raw specific humidity values in "
             "the underlying met data is deprecated."
         )
+    if isinstance(compute_tau_cirrus, str) and compute_tau_cirrus != "auto":
+        raise ValueError(
+            "Parameter ``compute_tau_cirrus`` must be one of"
+            f" True, False, or 'auto'. Found {compute_tau_cirrus}."
+        )
     if "tau_cirrus" not in met.data:
         met.ensure_vars(Cocip.met_variables)
-        met.data["tau_cirrus"] = tau_cirrus.tau_cirrus(met)
+        if (
+            compute_tau_cirrus == "auto"
+            and met.data["specific_humidity"].chunks is not None
+            or isinstance(compute_tau_cirrus, bool)
+            and compute_tau_cirrus
```

Comment:
I had to think about this for a minute ... I thought there could be someway to simplify this clause before coming around to the way you have it.

Reply:
Yeah, I spent embarrassingly long thinking about this - sometimes Python implicit bool casting is not your friend :)